kayvee.go

```diff
@@ -19,7 +20,11 @@ const (
 
 // Format converts a map to a string of space-delimited key=val pairs
 func Format(data map[string]interface{}) string {
-	formattedString, _ := json.Marshal(data)
+	formattedString, err := json.Marshal(data)
+	if err != nil {
+		log.Printf("Error converting kayvee message to json: %s", err.Error())
```

Comment:
Yeah, it would work, I just feel weird about having a function call itself recursively when it errors. Generally, I feel weird about having something rely on itself when it fails, but maybe that's silly of me given that we know this will work. I'll change.